misc/acrn-config/xmls/config-xmls/generic/hybrid.xml

```diff
@@ -77,7 +77,6 @@
         <bootargs desc="Specify kernel boot arguments">
         rw rootwait console=tty0 consoleblank=0 no_timer_check quiet loglevel=3
         i915.nuclear_pageflip=1 i915.avail_planes_per_pipe=0x01010F i915.domain_plane_owners=0x011111110000 i915.enable_gvt=1
-        hvlog=2M@0x1fe00000 memmap=0x200000$0x1fe00000
```

Comment:
Does this part (If user want to support hvlog in his own board, then he can add
the param in sos cmdline manually) covered by the document already?

Reply:
not yet. It is in my TODO list.
Actually generic board should be the minimal subset of feature that we support. We should not cover memmap address info since it is board specific.

Reply:
Anyway, we need to describe it somewhere in the document. It could be done later.